posthog-react-native/src/posthog-rn.ts

```diff
@@ -22,6 +22,7 @@ export type PostHogOptions = PosthogCoreOptions & {
     | PostHogCustomAppProperties
     | ((properties: PostHogCustomAppProperties) => PostHogCustomAppProperties)
   customAsyncStorage?: PostHogCustomAsyncStorage
+  captureNativeAppLifecycleEvents?: boolean
```

Comment:
I feel like the naming here clashes with the PostHogAutocaptureOptions. No good option comes to mind but it feels like we might get messier and messier over time... I wonder if we shouldn't just include them as part of the "lifecycle" events, even though they're not strictly dependent on react lifecycles...

Reply:
I wanted it to be a separate flag so as to not spam people who update. We could definitely not do that and have it under the same flag, but as you mentioned above we would need to be careful about how we communicate the change, whereas with a new flag it's less of a big deal.